packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -735,22 +736,18 @@ export default {
             this.genericErrorMessage = messageToDisplay || this.$t('errorMessages.genericServerError');
 
             this.$nextTick(() => {
-                this.scrollToElement('errorAlert');
+                this.scrollToElement(this.$refs.errorAlert.$el);
             });
         },
 
         /**
          * Scroll to a ref element in the screen.
         */
-        scrollToElement (refElement) {
+        scrollToElement (element, options = { offset: -20 }) {
```

Comment:
Small thing – would be good to update the comment for this function to describe the function params. It's not obvious why the default of offset would be -20 for instance, whereas a comment would describe that.